README.md

```diff
@@ -163,7 +163,7 @@ optional arguments:
                         considered the library's source.
   --read-orientation-min-mapped-reads INT
                         minimum number of mapped reads for deeming the read
-                        orientation result reliable (default: 20)
+                        orientation result reliable (default: 18)
```

Comment:
Hmmm, I think 18 is so random that it gives a suggestion that we somehow determined this to be optimal in some way. I'd rather leave the default as it was and modify the test file such that it will have alignments for all 20 reads. Or alternatively set the param to 10 for the failing test.

Reply:
Yeah you're right, sorry, I changed back the default to 20 and updated the tests with the parameter.